.github/workflows/continuous_integration.yml

```diff
@@ -0,0 +1,113 @@
+name: Continuous Integration
+
+on:
+  push:
+    branches:
+      - master
+      - release
+      - 'v3.13-dev'
+  pull_request:
+    branches:
+      - master
+      - release
+      - 'v3.13-dev'
+
+jobs:
+  build-windows:
+    runs-on: windows-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Build and Test
+      run: .\build.ps1 --target=Test --test-run-name=Windows
+
+    - name: Package
+      run: .\build.ps1 --target=Package
+
+    - name: Upload build artifacts
+      uses: actions/upload-artifact@v2
+      with:
+        name: Package
+        path: package
+
+    - name: Upload test results
+      uses: actions/upload-artifact@v2
+      with:
+        name: Test results (Windows)
+        path: test-results
+      # Use always() to always run this step to publish test results when there are test failures
+      if: ${{ always() }}
```

Comment:
`if: always()` has been working for me.

Looks like the `${{` escape is only needed if you aren't already implicitly in an expression context or if you have operators: https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#about-contexts-and-expressions

Reply:
I took this directly from the GitHub docs, but good to know.